index.php

```diff
@@ -95,7 +95,7 @@ private function build_index() {
 
 			$uri = trim( sprintf( 
 					'%s/%s',
-					trim( $uri_info['dirname'], '/' ),
+					trim( $uri_info['dirname'], '/\\' ),
```

Comment:
Yes, as well as here
https://github.com/Pilskalns/orbiter/blob/cc1eeefaed59ebf81732d5d9278c3f0dcd23c6c7/index.php#L169
and here
https://github.com/Pilskalns/orbiter/blob/cc1eeefaed59ebf81732d5d9278c3f0dcd23c6c7/plugins/meta/meta.plugin.php#L44

\+ on last [commit](https://github.com/Pilskalns/orbiter/commit/f1c003582ee348ff7290b6fcf51b1fc021c383b9) there is fix to one more issue when md file options are not read as options, but instead just as plain text. It's better to use twice `PHP_EOL.PHP_EOL` instad of `"\n\n"` as newlines per system / per encoding / per code editor vary very differently. When file encoding goes really bad, even `PHP_EOL` don't work, then better solution is unify all chances of bad line breaks. I use something like this 

```php
	//Universal shit, will replace weird line brakes with regular ones
	$progress = str_replace(array('\r\n','\n\r','\r','\n',chr(10)),'\n',$progress);
```
Trust me on this, I have spend days fixing i/o issues on different software...